content/applications/hr/referrals/rewards.rst

```diff
@@ -0,0 +1,76 @@
+=======
+Rewards
+=======
+
+After employees have successfully earned referral points, they can exchange their points by
+purchasing rewards in Odoo's *Referrals* application. Rewards **must** be :ref:`created and
+configured <referrals/create>` before employees can :ref:`redeem points for rewards
+<referrals/redeem>`.
+
+.. _referrals/create:
+
+Create rewards
+==============
+
+Rewards are the only configurations needed when setting up the *Referrals* application. Only users
+with :guilabel:`Administrator` rights for the *Recruitment* application can create or modify
+rewards. To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`.
+Click :guilabel:`New`, and a reward form loads. Enter the following information on the form:
```

Comment:
```suggestion
Rewards are the only configurations needed when setting up the *Referrals* application.

Only users with :guilabel:`Administrator` rights for the *Recruitment* application can create or
modify rewards.

To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`. Click
:guilabel:`New`, and a reward form loads. Enter the following information on the form:
```
SUGGESTION to break up this otherwise "lengthy" paragraph to make it a bit more digestible to the reader.